config/initializers/good_job.rb

```diff
@@ -3,6 +3,7 @@
     config.good_job.execution_mode = :external
     config.good_job.poll_interval = 5
     config.good_job.enable_cron = true
+    config.good_job.queues = 'default:5,low:1;*'
```

Comment:
More info on queues syntax here: https://github.com/bensheldon/good_job#optimize-queues-threads-and-processes

Reply:
this ratio was a complete guess